src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,145 @@
 package core.basesyntax;
 
+import static java.util.Objects.hash;
+
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int MAXIMUM_CAPACITY = 1073741824;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final int GROW_CAPACITY_FACTOR = 2;
+
+    private int size;
+    private int threshold;
```

Comment:
not worth to keep it as a field, we cal always calculate it from capacity and threshold (constant)